app/controllers/packages_controller.rb

```diff
@@ -1,7 +1,27 @@
 class PackagesController < ApplicationController
+  before_action :set_event, only: [:index, :show]
+  before_action :set_package, only: [:show]
+
   def index
+    @packages = Package.all?
```

Comment:
no need to close this pull but wondering why you're checking the Package.all? instance with the question mark... what exactly are you confirming?

Reply:
@Matth93